R/lasso-deployment.R

```diff
@@ -7,16 +7,14 @@
 #' \item Push these predictions to SQL Server
 #' }
 #' @docType class
-#' @usage LassoDeployment(type, df, grainCol, testWindowCol, predictedCol, 
-#' impute, debug)
+#' @usage LassoDeployment(type, df, grainCol, predictedCol, impute, debug, cores, modelName)
 #' @import caret
 #' @import doParallel
 #' @importFrom R6 R6Class
 #' @import ranger
 #' @param type The type of model (either 'regression' or 'classification')
 #' @param df Dataframe whose columns are used for calc.
 #' @param grainCol The dataframe's column that has IDs pertaining to the grain
-#' @param testWindowCol (depreciated) All data now receives a prediction
 #' @param predictedCol Column that you want to predict. If you're doing
 #' classification then this should be Y/N.
 #' @param impute For training df, set all-column imputation to F or T.
```

Comment:
Let the user know that impute uses the values from the model that was saved in the development step. Do this for all 4 deployment docs